schedule_lessons/dashboard/static/dashboard/css/agenda.css

```diff
@@ -395,48 +416,44 @@ html body {
     margin-top: -125px;
     display: none;
   }
-
   .lesson {
     width: 95% !important;
   }
 }
 
 @media (max-width: 991.98px) {
-
   #loggedInNavbar .navbar-brand {
     font-size: 180%;
     font-family: AvenirNext-DemiBold;
     padding-top: 0.15%;
     padding-bottom: 0.15%;
     margin-right: auto;
   }
-
   #navbarContent .nav-link {
     text-decoration: none;
-    color: rgba(255,255,255,.5);
+    color: rgba(255, 255, 255, .5);
   }
-
   #navbarContent .nav-link:hover {
-    color: rgba(255,255,255,.75);
+    color: rgba(255, 255, 255, .75);
   }
-
   #loggedInNavbar .active .nav-link {
     color: white;
   }
 }
+
 /* END Navbar for logged in Users */
 
 #picturePrompt {
-  height: 300px;
+  width: 90%;
```

Comment:
I am guessing you made this change to look better on mobile, but the image was too huge on laptop. Please check out my commit on "Fixing the huge picture prompt" and let me know if the changes I made look alright